px-vis-register-item.html

```diff
@@ -67,10 +67,24 @@
         class$="flex [[_itemMutedToStart]]"
         on-tap="_seriesClicked"
         id="regWithoutMenu">
-        <span
-          class="seriesMarker"
-          style$="background-color:[[_itemColor]]"
-          name$="R{{item.name}}">&nbsp;</span>
+        <template is="dom-if" if="[[_isDashed(_dashPattern)]]">
```

Comment:
Similar to the access marker I left the existing in place and just added the optional svg line marker.